swagger.json

```diff
@@ -4074,6 +4238,25 @@
                 "nl"
             ]
         },
+        "Title": {
+            "properties": {
+                "title": {
+                    "type": "string",
+                    "description": "The title of an item",
```

Comment:
Misschien beter om hier OrganizerTitle, of OrganizerName (zie bovenstaande opmerking) van te maken en in de description "item" te vervangen met "organizer", omdat organizer denk ik de enige is waar de title/name (nog) niet vertaalbaar is en dus een gewone string is.